local-volume/README.md

```diff
@@ -102,30 +102,73 @@ $ ALLOW_PRIVILEGED=true LOG_LEVEL=5 FEATURE_GATES=PersistentLocalVolumes=true ha
 
 ### Step 2: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Deploying the external static local provisioner 
 
-This is optional, only for automated creation and cleanup of local volumes. See
-[bootstrapper/](./bootstrapper) and [provisioner/](./provisioner) for details and
-sample configuration files.
+This option uses **helm** templating to generate the provisioner's DaemonSet and ConfigMap specs.
+The generated specs can be further customized as needed (usually not necessary), and then deployed using kubectl.
+
+To install helm's client interface, run the following two command:
 
-1. Create an admin account with cluster admin priviledge:
```

Comment:
We need to keep the step for creating a service account for the daemonset

Reply:
done